src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultDisplayTemplates.cs

```diff
@@ -224,9 +224,15 @@ public static string ObjectTemplate(IHtmlHelper htmlHelper)
 
             var serviceProvider = htmlHelper.ViewContext.HttpContext.RequestServices;
             var viewEngine = serviceProvider.GetRequiredService<ICompositeViewEngine>();
-            var properties = modelMetadata.Properties.Where(metadata => ShouldShow(metadata, templateInfo));
-            foreach (var propertyMetadata in properties)
+
+            foreach (var propertyExplorer in modelExplorer.Properties)
```

Comment:
believe I suggested change from `metadata.Properties` to `modelExplorer.Properties`.  sorry if so.  we only want properties of declared type here and in editor equivalent.